spec/controllers/verify/finance_controller_spec.rb

```diff
@@ -141,6 +141,8 @@
 
     context 'when the form is invalid' do
       it 'tracks the form errors and does not make a vendor API call' do
+        expect(Idv::FinancialsValidator).to_not receive(:new)
```

Comment:
this is minor (and possibly inconsistent with the style of our test suite) but I usually find it easier to read a spec when all expectations are together. 

So here I would spy on this in the spec setup: `allow(Idv::FinancialsValidator).to receive(:new)` and then below with the other expectations I would include `expect(Idv::FinancialsValidator).not_to have_received(:new)`